content/work/medcurbside.md

```diff
@@ -54,5 +54,56 @@ summary: |
   in healthcare today.
 ---
 
+{% import 'embed.macros.njk' as embed %}
+{% import 'layout.macros.njk' as layout %}
 {% import 'quotes.macros.njk' as quotes %}
+
+{{ layout.title('What We Did') }}
+
+{% call embed.media_block(
+  media=embed.screen(
+    notch='0',
+    content=embed.img(
+      src='work/medcurbside/medcurbside-phone.jpg',
+      alt='clinician questions with one answer and one source cited',
+      sizes='media'
+    )
+  )
+) %}
+
+{% call layout.grid('narrow-columns') %}
+
+{% call layout.block('column') %}
+
+### Services
+
+- Research & Concepting
+- Project Management
+- UX & UI Design
+- Front-end Development
+- Back-end Development
+- Ongoing Maintenance
+
+
+
+{% endcall %}
+
+{% call layout.block('column') %}
+
+### Languages & Tools
+
+- Adobe Xd 
+- Django & Python
+- Herman
+- JavaScript
```

Comment:
I imagine we used a framework of some kind on this?

Reply:
@jgerigmeyer @stacyk Is there a framework that could be added? (see Mia's comment)